src/ConnectedMode/ServerSentEvents/SSESessionManager.cs

```diff
@@ -27,16 +27,9 @@
 
 namespace SonarLint.VisualStudio.ConnectedMode.ServerSentEvents
 {
-    /// <summary>
-    /// Reacts to project changes and opens/closes Server Sent Events sessions
-    /// </summary>
-    internal interface ISSESessionManager : IDisposable
-    {
-    }
-
-    [Export(typeof(ISSESessionManager))]
+    [Export(typeof(SSESessionManager))]
```

Comment:
We dont use this anywhere else so I removed the interface so it is the same as the TriggerImportInstall class